src/core/utils.py

```diff
@@ -0,0 +1,23 @@
+"""Utils.
+
+This module provides classes to represent utils.
+"""
+
+import yaml
+
+
+class FileUtil:
+    """
+    File Util class represents methods working with files.
+    """
+
+    @staticmethod
+    def read_yaml_file(file_path: str):
```

Comment:
@vasylboliuk please don't forget to add annotations for the returning type

Reply:
Done